content/articles/packet-filtering-using-acls/index.md

```diff
@@ -0,0 +1,172 @@
+
+For instance, we have created ACL 10 which permits a particular host to the internet.
+```bash
+R1(config)# access-list 10 remark ACE permits ONLY host 192.168.10.10 to the internet
+R1(config)# access-list 10 permit host 192.168.10.10
+R1(config)# do show access-lists
+Standard IP access list 10
+    10 permit 192.168.10.10
+R1(config)#
+```
+Lets consider the  Local Area Network below:
+![CDP-LLDP Network Configuration](/engineering-education/packet-filtering-using-acls/acl.jpg)
+
+We will create one numbered ACL, ACL 20 that denies host 192.168.10.10 but permits all other hosts on network 192.168.10.0/24.
+
+Start by configuring the ACL 20 ACE that denies the 192.168.10.10 host using the `host` keyword, then create the ACL that permits all other hosts.
+```bash
+R1(config)#access-list 20 deny host 192.168.10.10
+R1(config)#access-list 20 permit 192.168.10.0 0.0.0.255
+```
+Since ACL 20 only apply to traffic from LAN 1, the ACL can be applied to the incoming traffic to the G0/0/0 R1's interface.
+Enter interface g0/0/0 mode, apply ACL 20 inbound and return to global configuration mode.
+```bash
+R1(config)#interface g0/0/0
+R1(config-if)#ip access-group 20 in
+R1(config-if)#exit
+```
+We will then create a named standard ACL that permits host 192.168.10.10 but denies all other hosts on network 192.168.20.0/24.
+Start by creating a named standard ACL named LAN2-FILTER
+```bash
+R1(config)#ip access-list standard LAN2-FILTER
+
+```
+Create an ACE that permits host 192.168.10.10, and deny all other hosts using the `any` keyword.
+```bash
+R1(config-std-nacl)#permit host 192.168.10.10
+R1(config-std-nacl)#deny any
+R1(config-std-nacl)#exit
+```
+The LAN2-FILTER will be applied to the outgoing traffic to LAN 2.
+Enter interface g0/0/1 mode, apply ACL LAN2-FILTER outbound and return to global configuration mode.
+```bash
+R1(config)#interface g0/0/1
+R1(config-if)#ip access-group LAN2-FILTER out
+R1(config-if)#exit
+```
+>NOTE:  After creating and configuring the ACLs you can use the `show access-lists` or `show run | section ip access-list` commands in the priviledge exec mode to verify your configurations.
+
+>To remove an ACLs use the `no access-list` command followed by the name or number of the access list. For instance `no access-list 10` to remove ACL 10.
+
+### Conclusion
+As we have seen, Access Control Lists play a crucial role in traffic flow control and the network's security at large. This makes the network less vulnerable to unwanted and dangerous traffic.
+To summarize, we have:
+- Learned what Access Control Lists are, their operations, and how to create and configure them.
+- Explored different types of ACLs used in packet filtering.
+- Configured a network, created and configured the standard named and numbered ACLs.
+- Learned how the configured ACLs can be used to permit or deny traffic.
+
+One can find more information about network management [here](https://contenthub.netacad.com/ensa/5.0.1).
+
```

Comment:
This link doesn't seem to work. Please provide a valid link.

Reply:
Ook I will